frontend/src/v5/ui/v4Adapter/overrides/properties.overrides.ts

```diff
@@ -67,8 +67,8 @@ export default css`
 
 	// Drop-down inputs
 	${StyledFormControl} {
-		label { left: -14px; }
-		.MuiInput-root svg { margin-top: 23px; }
+		/* label { left: -14px; } */
```

Comment:
If you don't need it anymore, could you please remove this comment?

Reply:
oops